pypeman/plugins/remoteadmin/views.py

```diff
@@ -0,0 +1,241 @@
+import json
+import logging
+
+from aiohttp import web
+from jsonrpcserver.response import SuccessResponse
+
+from pypeman import channels
+
+logger = logging.getLogger(__name__)
+
+
+def get_channel(name):
+    """
+    return channel by is name.all_channels
+    """
+    for chan in channels.all_channels:
+        if chan.name == name:
+            return chan
+    return None
+
+
+async def list_channels(request, ws=None):
+    """
+    Return a list of available channels.
```

Comment:
Try to document al little more.
So if I understand:

- User preforms a get request to /channels but doesn't get the responde directly but only via a web socket?
This is not obvious (at least for me)
- how does this work on the client side.
- why a web socket is this really a slow operation or an operation where one wants to push something or is the idea
- that a debug interface would 

SO how DO I have to use this?